src/_locales/tr/messages.yml

```diff
@@ -0,0 +1,420 @@
+  message: Güncellemeler denetlemiyor...
+msgErrorFetchingUpdateInfo:
+  description: Message shown when Violentmonkey fails fetching version data of the script.
+  message: Bilgileri güncellerken hata oluştu.
+msgNoUpdate:
+  description: Message shown when there is no new version of a script.
+  message: Güncelleme bulunamadı.
+labelInstall:
+  description: Shown in the title of the confirm page while trying to install a script.
+  message: Script yükleniyor
+buttonInstallOptions:
+  description: Button to show options of installation confirm page.
+  message: Ayarlar
+installOptionClose:
+  description: Option to close confirm window after installation.
+  message: Yüklemeden sonra kapat
+installOptionTrack:
+  description: Option to track the loading local file before window is closed.
+  message: Bu pencere kapanmadan önce yerel dosyayı takip et
+buttonConfirmInstallation:
+  description: Button to confirm installation of a script.
+  message: Yüklemeyi onayla
+buttonClose:
+  description: Button to close window.
+  message: Kapat
+msgErrorLoadingData:
+  description: >-
+    Message shown on confirm page when the script to be installed cannot be
+    loaded.
+  message: Script verisi yüklenirken hata oluştu.
+msgLoadedData:
+  description: >-
+    Message shown in the confirm page when a javascript file to be installed is
+    loaded.
+  message: Script verileri yüklendi.
+msgErrorLoadingDependency:
+  description: Message shown when not all requirements are loaded successfully.
+  message: Script yüklemesinde veriler tam yüklenmedi.
+msgLoadingDependency:
+  description: Message shown on confirm page when the requirements are being downloaded.
+  message: Veriler yükleniyor... ($1/$2)
+msgLoadingData:
+  description: Message shown on confirm page when the script to be installed is loading.
+  message: Script verileri yükleniyor...
+sideMenuInstalled:
+  description: 'Side menu: Installed scripts'
+  message: Yüklenen Scriptler
+sideMenuSettings:
+  description: 'Side menu: Settings'
+  message: Ayarlar
+sideMenuAbout:
+  description: 'Side menu: About'
+  message: Hakkında
+buttonNew:
+  description: Button to create a new script.
+  message: Yeni
+buttonUpdateAll:
+  description: Check all scripts for updates.
+  message: Yeni güncellemeleri kontrol et
+anchorGetMoreScripts:
+  description: Link to get more scripts.
+  message: Daha fazla script
+  touched: false
+msgLoading:
+  description: Message shown in the options page before script list is loaded.
+  message: Yükleniyor ...
+labelSettings:
+  description: Label shown on the top of settings page
+  message: Ayarlar
+labelAutoUpdate:
+  description: Option to allow automatically checking scripts for updates every 24 hours.
+  message: Scriptleri her gün otomatik olarak kontrol et
+labelIgnoreGrant:
+  description: Option to suppress no @grant warnings.
+  message: `@grant` izin bulunmama uyarılarını engelle
```

Comment:
Still invalid, strings cannot start with `@` or `` ` `` in yaml, please wrap them in quotes. (`'...'`)